icons/align-justify.svg

```diff
@@ -9,8 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <line x1="21" y1="10" x2="3" y2="10" />
```

Comment:
This doesn't feel like "justify" to me.

Reply:
other option:
![image](https://user-images.githubusercontent.com/11825403/97358778-0737e300-189c-11eb-8651-87d12c9615f3.png)

I can also fill it up but then it will be the exact icon as the menu icon.

Reply:
Think it's best to "fill it up"

Reply:
Then it's the same as the menu-icon, I think we can better remove this one and add an tag for it.

Reply:
I stretched it